additionalCSS/workSection.css

```diff
@@ -0,0 +1,123 @@
+.workTextContainer {
+  display: flex;
+  justify-content: center;
+}
+
+.workText {
+  font-family: "Inter", sans-serif;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 40px;
+  text-align: center;
+  line-height: 52px;
+  padding-top: 22px;
+  width: 70%;
+}
+
+.indicator {
+  display: flex;
+  justify-content: center;
+  margin-top: 24px;
+}
+
+.indicatorImg {
+  width: 48px;
+  height: 4px;
+  border-radius: 24px;
+  background: #36b37e;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.workContainer {
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: repeat(5, 474px);
+  row-gap: 24px;
+  justify-content: center;
+  padding-top: 62px;
+}
+
+.firstContainer {
+  border-radius: 8px;
+  background: #ebf0ee;
+  outline: 1px solid #d0d9d4;
+}
+
```

Comment:
- [ ] As suggested by the previous reviewer please use the property `display: flex` on your `.firstContainer` class as according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) you must align the content inside each project card using `flexbox`

<hr>

>  - In order to lay out the elements in "Works" section you should use CSS `Grid `and `Flexbox `combined (`it is mandatory that you use both`). Use CSS `Grid `in the `main container` that contains all of the `cards `and use` Flexbox` to lay out an` individual card`.

<hr>